src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
I have asked because the RtlpGetFunctionEndAddress for ARM and ARM64 looks as follows, so it clearly has encoded function length in there.
```c++
ULONG
RtlpGetFunctionEndAddress (
    __in PT_RUNTIME_FUNCTION FunctionEntry,
    __in ULONG ImageBase
    )
{
    ULONG FunctionLength;
    
    FunctionLength = FunctionEntry->UnwindData;
    if ((FunctionLength & 3) != 0) {
        FunctionLength = (FunctionLength >> 2) & 0x7ff;
    } else {
        FunctionLength = *(PTR_ULONG)(ImageBase + FunctionLength) & 0x3ffff;
    }
    
    return FunctionEntry->BeginAddress + 2 * FunctionLength;
}

```

And the CEEJitInfo::allocUnwindInfo sets it like this:
```C++
size_t unwindInfoDeltaT = (size_t) pUnwindInfo - baseAddress;
...
RUNTIME_FUNCTION__SetUnwindInfoAddress(pRuntimeFunction, unwindInfoDelta);
```